src/highs.jl

```diff
@@ -5,15 +5,16 @@ using HiGHS: Highs_create, Highs_setStringOptionValue, Highs_setSolution,
              Highs_getModelStatus, HighsInt
 
 function solve_IP(::Val{:highs}, O::OptProblem, initial_solution = Int[],
-                  use_warmstart = true; options...)
+                  use_warmstart = true; solver_options, options...)
     model = Highs_create()
-    Highs_setStringOptionValue(model, "mip_rel_gap", "0")
```

Comment:
The gaps are somewhat problematic. They won't cause incorrect results but may interfere with convergence to optimality. Now that's by design what they are for, but should probably rather be specified for the minimal feedback arc set problem itself and then be converted to solver gaps. It's not clear how to do that though, so I'll have to think about it.

Reply:
I think this is best left as a solver option for the user with some default that works well for the problem generally.